Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -577,28 +577,41 @@ static RefPtr<CSSValue> positionOffsetValue(const RenderStyle& style, CSSPropert
     return CSSPrimitiveValue::create(CSSValueAuto);
 }
 
+RefPtr<CSSValue> ComputedStyleExtractor::textWrapShorthandValue(const RenderStyle& style) const
+{
+    auto textWrapMode = style.textWrapMode();
+    auto textWrapStyle = style.textWrapStyle();
```

Comment:
I like taking advantage of scope to have short variable names. I would just use `mode` and `style` here, because in this context that’s enough.

Reply:
I had `mode` / `style` originally, but `style` conflicts with the parameter name.